examples/Cargo.toml

```diff
@@ -21,3 +21,11 @@ tokio = { workspace = true }
 [[example]]
 name = "user-operation"
 path = "user_operation.rs"
+
+
+[dependencies]
+jsonrpsee = {version = "0.18.2", features = ["server", "http-client"]}
```

Comment:
Is not formatted error is because features are not sorted and there is no whitespace between `{version` -> `{ version` and `"http-client"]}` -> `"http-client"] }`

Reply:
Thanks for pointing it out! 

I've added `cargo sort --workspace` command to the `Makefile` in the latest commit. Otherwise, it's a pain in the ass to sort & format deps manually.

Reply:
Yeah, that makes sense, cool